packages/circuit-ui/cli/migrate/__tests__/migrate.spec.js

```diff
@@ -54,6 +54,9 @@ defineTest('theme-border-radius');
 defineTest('button-default-size');
 defineTest('label-prop-names');
 
+// v4
+defineTest('icons-v2');
```

Comment:
I kept the transforms in a single codemod, thought it was not helpful to split and make developers run several of them, especially since the transforms are so closely related